legal-api/tests/unit/resources/v2/test_business.py

```diff
@@ -365,17 +366,22 @@ def test_post_affiliated_businesses(session, client, jwt):
                                legal_type=business[1])
 
     for draft_business in draft_businesses:
-        filing_name = 'incorporationApplication' if draft_business[1] == Business.LegalTypes.BCOMP.value else 'registration'
+        filing_name = draft_business[1]
         temp_reg = RegistrationBootstrap()
         temp_reg._identifier = draft_business[0]
         temp_reg.save()
         json_data = copy.deepcopy(FILING_HEADER)
         json_data['filing']['header']['name'] = filing_name
         json_data['filing']['header']['identifier'] = draft_business[0]
-        json_data['filing']['header']['legalType'] = draft_business[1]
-        if draft_business[2]:
+        json_data['filing']['header']['legalType'] = draft_business[2]
+        if draft_business[3]:
             json_data['filing'][filing_name] = {
-                'nameRequest': {'nrNumber': draft_business[2]}
+                'nameRequest': {'nrNumber': draft_business[3]}
+            }
+        if filing_name == 'amalgamation':
+            json_data['filing'][filing_name] = {
+                **json_data['filing'][filing_name],
+                'amalgamationType': 'regular'
```

Comment:
`amalgamationType` should be `type`

Reply:
That's a FE bug :)

I will fix it in Create UI.

Reply:
Fixed as 'type`